src/getdns/getdns.h.in

```diff
@@ -46,7 +46,10 @@ extern "C" {
 
 #define GETDNS_COMPILATION_COMMENT "@GETDNS_COMPILATION_COMMENT@"
 
-/* Return values */
+/**
+ * \defgroup returntypes Return types
```

Comment:
> On 22 Jun 2016, at 12:48, wtoorop <notifications@github.com> wrote:
> 
> In src/getdns/getdns.h.in <https://github.com/getdnsapi/getdns/pull/186#discussion_r68038731>:
> 
> > @@ -46,7 +46,10 @@ extern "C" {
> >  
> >  #define GETDNS_COMPILATION_COMMENT "@GETDNS_COMPILATION_COMMENT@"
> >  
> > -/* Return values */
> > +/**
> > + * \defgroup returntypes Return types
> Return types? Not return values?
> 
Hi Willem,

What I did was for consistency was call all those constants that are defined within a type (via typedef enum {}) as a ‘Type' and those that are straight #defines ‘Values'. This maybe too implementation specific, but the previous naming seemed a bit arbitrary to me. I don’t mind too much what we decide but I do like consistency….

Sara.